src/tests/nativeaot/SmokeTests/UnitTests/Generics.cs

```diff
@@ -53,7 +53,10 @@ internal static int Run()
         TestNullableCasting.Run();
         TestVariantCasting.Run();
         TestMDArrayAddressMethod.Run();
-        TestNativeLayoutGeneration.Run();
+// SIGSEGV handler missing: https://github.com/dotnet/runtime/issues/87458
+#if !TARGET_IOS && !TARGET_TVOS
```

Comment:
I tried to disable failing tests using `#ifdef`, but it didn't work. Could it be that the syntax is incorrect?

Reply:
I am not 100% sure but I believe that: `TARGET_IOS`, `TARGET_TVOS`, etc are not defined in `*.props`, `*.targets` or `*.*proj` files that are referencing this file. 

Which means that, there should be somewhere in those files something like:
```xml
<DefineConstants Condition="'$(TargetsiOS)' == 'true'">$(DefineConstants);TARGET_IOS</DefineConstants>
```
for example.